WinUIConvert.md

```diff
@@ -0,0 +1,101 @@
+# Converting To WinUI3 Using Try-Convert
+
+This document details how to use the Try-Convert Tool to convert a C# UWP App to WinUI3 using .NET Native.
+
+## Introduction
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+- WinUI2 is an official library with support for native Windows UI elements for Windows apps. 
+- WinUI3 Update now supports both Windows Desktop and UWP apps. 
+
+Updating a C# UWP App to WinUI3 can be an involved process and requires several changes to the .csproj as well as C# code. Running this tool will automate the process. 
+
+## Description
+This tool assists with the conversion process by first modifying .csproj files to use the new `Microsoft.WinUI` nuget package. It converts nuget packages which are incompatible with the new WinUI3 where possible and removes them otherwise and converts
+
+### .csproj File Examples
+Highlighting changes not entire proj file () witre this better
+
+#### .csproj Before Running Try-Convert
+```xml
+<ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" />
+    <!-- These packages no longer work -->
+    <PackageReference Include="ColorCode.UWP" />
+    <PackageReference Include="Win2D.uwp" />
+    <!-- This Needs to be Converted-->
+    <PackageReference Include="Microsoft.UI.Xaml" />
+</ItemGroup>
+```
+#### .csproj After Running Try-Convert
+```xml
+<ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" />
+    <PackageReference Include="Microsoft.WinUI" />
```

Comment:
What happens to the `ColorCode.uwp` and `Win2D.uwp` packages?

Reply:
@stevenbrix 
I updated the .md for Microsoft.Win2D.UWP to be converted. ColorCode.uwp would be removed. Is removing the package reference incorrect or is there a replacement for ColorCode.uwp?

Reply:
i'd ask Alex about colorcode, but we can follow up with that later

Reply:
There is a ColorCode.WinUI but, as of right now, its only in our MyGet:
https://dotnet.myget.org/feed/uwpcommunitytoolkit/package/nuget/ColorCode.WinUI

Reply:
Added uwp comunitytoolkit versions